### PR TITLE
Update opensesame.rb to v3.1.0

### PR DESCRIPTION
**Pull request by mperreir**
opensesame was updated to v3.1.0 and download url is now correct.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/opensesame.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
     sha256 'b2a37cfd1c514b2ae8ddd0be09a274844420bfa432318ef87df308fdd3b6a770'
     url "http://files.cogsci.nl/software/opensesame/opensesame_#{version}-macos-2.zip"
   else
-    version '0.27.4'
-    sha256 '8814da8fe5e638cb7db18b4e8188fc97028bd98f1603ceae006aff13745fc739'
-    url "http://www.cogsci.nl/dschreij/opensesame-mac/opensesame-#{version}-macos-x86_64-1.dmg"
+    version '3.1.0'
+    sha256 ''
```
Comment (vitorgalvao) [inline Casks/opensesame.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/opensesame.rb [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
+    url "https://github.com/smathot/OpenSesame/releases/download/release%2F#{version}/opensesame_#{version}-py2.7-macos-2.dmg"
```
Comment (reitermarkus) [inline Casks/opensesame.rb]: `%2F` can be change to a `/`.

Also, this need a comment, as the download url does not match the homepage anymore.

```
# github.com/smathot/OpenSesame was verified as official when first introduced to the cask
```

Comment (vitorgalvao) [inline Casks/opensesame.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by mperreir** [comment]
- [ x ] Commit message includes cask’s name (and new version, if applicable).
- [ x ] `brew cask audit --download {{cask_file}}` is error-free.
- [ x ] `brew cask style --fix {{cask_file}}` left no offenses.

- sha hash updated and appcast added

**Comment by vitorgalvao** [comment]
Did you run the commands? Because Travis is failing.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1500beb5d9ca06d9657af6f80d0e13f047391bb1. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**